frontend/src/js/actions/Actions.js

```diff
@@ -67,7 +67,10 @@ var Actions = {
     initializeGraphicsMethodsValues(gm) {
         return {
             type: 'INITIALIZE_GRAPHICS_METHODS_VALUES',
-            graphics_methods: gm
+            graphics_methods: gm,
+            gmProps: gm['boxfill']['default'],
+            gmParent: 'boxfill',
+            gm: 'default'
```

Comment:
Yeah, this is the initial setup of the store's graphics_methods state. I'm removing the gmProps/Parent/gm as those are an artifact of when I was still learning how to pass state through the store, and completely not relevant.